common/src/main/java/com/microsoft/identity/common/internal/ui/webview/AzureActiveDirectoryWebViewClient.java

```diff
@@ -155,22 +158,66 @@ private boolean handleUrl(final WebView view, final String url) {
             return true;
         } else if (isRedirectUrl(formattedURL)) {
             Logger.info(TAG, "Navigation starts with the redirect uri.");
-            return processRedirectUrl(view, url);
+            processRedirectUrl(view, url);
+            return true;
         } else if (isWebsiteRequestUrl(formattedURL)) {
             Logger.info(TAG, "It is an external website request");
-            return processWebsiteRequest(view, url);
+            processWebsiteRequest(view, url);
+            return true;
         } else if (isInstallRequestUrl(formattedURL)) {
             Logger.info(TAG, "It is an install request");
-            return processInstallRequest(view, url);
+            processInstallRequest(view, url);
+            return true;
         } else if (isWebCpUrl(formattedURL)) {
             Logger.info(TAG, "It is a request from WebCP");
-            return processWebCpRequest(view, url);
+            processWebCpRequest(view, url);
+            return true;
+        } else if (isPlayStoreUrl(formattedURL)){
+            Logger.info(TAG, "Request to open PlayStore.");
+            return processPlayStoreURL(view, url);
+        } else if (isAuthAppMFAUrl(formattedURL)){
+            Logger.info(TAG, "Request to link account with Authenticator.");
+            processAuthAppMFAUrl(url);
+            return true;
+        } else if (isValidRedirectUri(url)){
```

Comment:
>isValidRedirectUri [](start = 19, length = 18)

if it is a valid redirect uri, why are we processing it as invalidRedirectUri. Did you mean to invert the condition here?

Reply:
Addressed here : https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/1195/commits/6f039fe2c3b99c4c6d497cea41431ba645475f83